bin/publish.sh

```diff
@@ -19,7 +19,13 @@ read -p "
 If you are missing the dev-a sub domain that means you will be publishing to PRODUCTION.
 
 
-Are you sure you're ready to publish the file prebid-universal-creative.js to $DOMAIN?" -n 1 -r
+Are you sure you're ready to publish following files:
+- prebid-universal-creative-amp.js
```

Comment:
Open to suggestions about filenames

Reply:
These are fine with me and in-line with our current naming convention.

When we test we just need to ensure these don't cause any issues as they will now be used as GAM key/values for substitutions.  If we run into issues we will need to shorten the names.